src/app/app.component.ts

```diff
@@ -68,3 +95,9 @@ export class AppComponent implements OnDestroy {
         }
     }
 }
+
+export enum Valid {
+    Error,
+    Warn,
+    Success,
+}
```

Comment:
_Nice to have_:

Ich bin mittlerweile ein Fan von Types, da man solche Sachen wie `public Valid = Valid;` dann nicht braucht um es im Template zu verwenden.
Du hast dann trotzdem auch Autovervollständigung und Co. aber brauchst diesen (ziemlich unschönen) Zwischenschritt nicht.

Ein weiterer Vorteil ist, dass du gleich einen "schönen String" hast, den du auch z.B. speichern kannst. Bei einer Enum, wo keine Werte zu gewiesen werden, ist alles basierend auf dem Index der Enum. Spielt bei uns weniger ne Rolle, aber wollte ich noch mit erwähnen.

Den export brauchst du vermutlich auch gar nicht, zumindest habe ich keine Verwendung außerhalb der Datei gesehen.

```suggestion
type ValidationStatus = 'error' | 'warn' | 'success'
```